searches/utils.py

```diff
@@ -42,3 +88,55 @@ def course_name_contains_token(token):
         | Q(name__icontains=token.replace("&", "and"))
         | Q(name__icontains=token.replace("and", "&"))
     )
+def query_is_abbreviation(query):
+    query = query.lower()
+    if query  in abbreviations:
+        return reduce(operator.or_, map(lambda n: Q(name=n), abbreviations[query]))
+    return None
+
+# def query_is_abbreviation(query):
```

Comment:
You can delete all of the commented out code - the changes are tracked in git anyways so we can always refer to them later